internal/deviations/deviations.go

```diff
@@ -301,6 +301,11 @@ func GNOIStatusWithEmptySubcomponent(_ *ondatra.DUTDevice) bool {
 	return *gNOIStatusWithEmptySubcomponent
 }
 
+// InterfaceConfigVrfBeforeAddress returns if vrf should be configured before IP address when configuring interface.
+func InterfaceConfigVrfBeforeAddress(_ *ondatra.DUTDevice) bool {
```

Comment:
while here, please clean this up to capitalize VRF in the function and flag name

Reply:
done